### DISCUSSION TITLE
Brazilian Portuguese (PT-BR) - final translation

### DISCUSSION THREAD

**Discussion by carlosandrerj** [aclap-dev/video-downloadhelper#1164] [Language & Translation]
Final translation for Brazilian Portuguese (PT-BR).
Excellent app, I'm glad to help!
Greetings from Rio, Brazil
[pt_BR.txt](https://github.com/user-attachments/files/16655300/pt_BR.txt)

**Comment by lcfrederico** [comment]
Here's my contribution.
[pt_BR.txt](https://github.com/user-attachments/files/17043829/pt_BR.txt)
